eya_def_tools/eya_def_tools/data_models/reference_wind_farm.py

```diff
@@ -344,15 +346,6 @@ class ReferenceWindFarm(EyaDefBaseModel):
             "should not be empty if the field is included."
         ),
     )
-    wind_farm_id: str = pdt.Field(
```

Comment:
The ID of the wind farm and the reference wind farm are the same. There is no need to use two separate values.